frontend/src/components/Body/MainBody.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import {bodyItems} from "./BodyItems";
+import "./MainBody.css"
+
+class Mainbody extends Component {
+  render () {
+    return (
+      <div className="main-part">
+        <h5 className="main-heading">paint your life colorful</h5>
+        <div className="main-component">
+          <div className="paint-grid">
+              {bodyItems.map((item, index) => {
+                return <div key={index}>
+                  <div className="grid-element" className={item.cName}>{item.title}{item.name}</div>
```

Comment:
Usage of index in key, is bad approach, consider using of  `name` as key

Reply:
Also when rendering items, it's better to present a Component here
              {bodyItems.map((item, index) => {
                <BodyItem ...props goes here  />
It can be reusable in future, separate styling, and naming of props can be changed to make it more readable